src/components/SiteFooter.js

```diff
@@ -0,0 +1,153 @@
+import React from "react";
+import {
+  AlternateList,
+  AlternateListItem,
+  CountySeal,
+  IconLink
+} from "@baltimorecounty/dotgov-components";
+
+import {
+  FindInformationItems,
+  PoliciesItems,
+  TranslateItems
+} from "./data/FooterData";
+
+const SiteFooter = props => {
+  return (
+    <footer className="dg_county-footer">
+      <div className="container">
+        <div className="row">
+          <div className="col-lg-6 col-md-12 col-sm-12">
+            <div className="dg_brand-footer">
+              <div className="dg_seal-container">
+                <CountySeal />
+              </div>
+              <h1>
```

Comment:
I think this needs to a p. I'm checking with @sdurphy to see if this is identical to teh header, if so we might want to create a component for it.